db/seeds/base/add_schedules.rb

```diff
@@ -8,21 +8,29 @@
   funding_cap = start_year >= 2024
   cohort = FactoryBot.create(:cohort, start_year:, funding_cap:)
   schedules.each do |schedule_identifier|
-    FactoryBot.create(:schedule, schedule_identifier, cohort:, change_applies_dates: false)
+    FactoryBot.build(:schedule, schedule_identifier, cohort:, change_applies_dates: false).save(validate: false)
   end
 end
 
 Cohort.where(start_year: 2025..).find_each do |cohort|
-  %i[
-    npq_ehco_december
-    npq_ehco_june
-    npq_ehco_march
-    npq_ehco_november
-    npq_leadership_autumn
-    npq_leadership_spring
-    npq_specialist_autumn
-    npq_specialist_spring
-  ].each do |schedule_identifier|
-    FactoryBot.create(:schedule, schedule_identifier, cohort:, change_applies_dates: false)
+  {
+    npq_ehco_december: 9,
+    npq_ehco_june: 9,
+    npq_ehco_march: 8,
+    npq_ehco_november: 9,
```

Comment:
I've totally guessed what these policy descriptors should be for EHCO - I've asked what they should be on the Jira ticket.